go/internal/feast/server/http_server.go

```diff
@@ -148,6 +149,17 @@ func (s *httpServer) getOnlineFeatures(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	statusQuery := r.URL.Query().Get("status")
+	status := false
+	if statusQuery != "" {
+		var err error
+		status, err = strconv.ParseBool(statusQuery)
+		if err != nil {
+			http.Error(w, fmt.Sprintf("Error parsing status query parameter: %+v", err), http.StatusBadRequest)
```

Comment:
This will not return the error response as json format.

Reply:
This is in order to conform with the rest of the code. None of the error responses in the go http server are in json format.

Reply:
We may need fix this in future.